x/auth/ante/fee.go

```diff
@@ -57,31 +59,53 @@ func (d DeductFeeDecorator) AnteDeps(txDeps []sdkacltypes.AccessOperation, tx sd
 			[]sdkacltypes.AccessOperation{
 				{
 					AccessType:         sdkacltypes.AccessType_READ,
-					ResourceType:       sdkacltypes.ResourceType_KV_BANK_BALANCES,
-					IdentifierTemplate: hex.EncodeToString(banktypes.CreateAccountBalancesPrefix(feeTx.FeePayer())),
+					ResourceType:       sdkacltypes.ResourceType_KV_AUTH_ADDRESS_STORE,
```

Comment:
we have unit tests for message types to issues, IRRC helped a bunch with Tony's refactor but it seems like it caught the ante handler issue. Could we add some unit tests for the antehandler code paths too?

Reply:
yup, will do

Reply:
Added unit tests